src/coreclr/vm/clsload.cpp

```diff
@@ -881,21 +675,12 @@ void ClassLoader::LazyPopulateCaseSensitiveHashTables()
         // (either images compiled with an old version of crossgen, or for case-insensitive type lookups in R2R modules)
         _ASSERT(pModule->IsReadyToRun());
 
-        EEClassHashTable * pNewClassHash = EEClassHashTable::Create(pModule, AVAILABLE_CLASSES_HASH_BUCKETS, FALSE /* bCaseInsensitive */, &amTracker);
+        EEClassHashTable * pNewClassHash = EEClassHashTable::Create(pModule, AVAILABLE_CLASSES_HASH_BUCKETS, NULL, &amTracker);
```

Comment:
Do you want this to be `PTR_EEClassHashTable`?

Reply:
No, SetAvailableClassHash takes a EEClassHashTable* and that's what we need there.